autogen/main/variables.tf.tmpl

```diff
@@ -306,13 +306,12 @@ variable "node_pools_oauth_scopes" {
 }
 {% endif %}
 
-{% if autopilot_cluster %}
 variable "network_tags" {
   description = "(Optional, Beta) - List of network tags applied to auto-provisioned node pools."
```

Comment:
Looks like this no longer needs to be marked as beta?

```suggestion
  description = "(Optional) - List of network tags applied to auto-provisioned node pools."
```

Reply:
Ack.